### ISSUE TITLE
aur-sync: add --install to automatically install new package after building it

### ISSUE THREAD

**Issue by terencode** [AladW/aurutils#304]
Right now, when installing a new package you have to do 2 actions:
1- <code>aur sync <i>package</i></code>
2- <code>sudo pacman -S <i>package</i></code>

With `-install` Aurutils would fetch the package, build it, add it to the repository database and install it right away.

**Comment by Earnestly** [comment]
It isn't `aurutils` job to install packages though, so this seems misplaced responsibility.

**Comment by maximbaz** [comment]
And again, if you want you can make your own wrapper `aur-sync-install` (something like below, but better), put it in `PATH` and use `aur sync-install <package>`.

```
aur sync $*
pacman -S $1
```

**Comment by terencode** [comment]
@maximbaz @Earnestly That makes sense to me. I'm just unaware of the way to proceed.
Maybe it would be useful to describe how to do it in a FAQ in the README?

**Comment by AladW** [comment]
I agree the documentation should be updated to reflect the new `aur(1)` design and how to drop-in your own scripts.

**Comment by AladW** [comment]
#217

**Comment by AladW** [comment]
The issue with the wrapper script https://github.com/AladW/aurutils/issues/304#issuecomment-367076222 is that it doesn't distinguish between `aur-sync` arguments, and package arguments. 

One could use any of the print options (`--no-build`, `--columns`, `--save`) but this does not distinguish between arguments (explicit installation reason) and their dependencies. Placing an `--install` flag into `aur-build` has the same issue.

**Comment by lamdevhs** [comment]
I actually like the behavior where packages built and added to the custom repository are not installed automatically in the system, yet this line from the latest release keeps installing every built package globally without even asking:
https://github.com/aurutils/aurutils/blob/f44785644347f83523958ed5f370c56aeec9fb03/lib/pacman/aur-build#L476
is this normal behavior? I just run `aur sync <package_name>` without any options...

**Comment by AladW** [comment]
This is done to install dependencies required for a build
